assets/js/modules/tabs.js

```diff
@@ -12,19 +11,6 @@ function scrollIntoView(el) {
     el.get(0).scrollIntoView(true);
 }
 
-function trackTabClick(label, trackTabClicksAsPageviews) {
```

Comment:
To the comment on the previous PR. I'm removing this because we never enable it anywhere (as tracking separate pageviews was confusing analytics reports). Plus removing it simplifies the tab macros which are a little complicated. Happy to leave this in you have objections.

Reply:
Ah I forgot it was no longer enabled. Kill it then!